packages/perseus/src/widgets/interactive-graphs/graphs/utils.test.ts

```diff
@@ -3,22 +3,7 @@ import {getIntersectionOfRayWithBox} from "./utils";
 import type {Interval, vec} from "mafs";
 
 describe("getIntersectionOfRayWithBox", () => {
-    it("returns [0, 0] when the given points are the same", () => {
-        const box: [Interval, Interval] = [
-            [-7, 7],
-            [-11, 11],
-        ];
-        const initialPoint: vec.Vector2 = [3, 5];
-        const throughPoint: vec.Vector2 = [3, 5];
-        const intersection = getIntersectionOfRayWithBox(
-            initialPoint,
-            throughPoint,
-            box,
-        );
-        expect(intersection).toEqual([0, 0]);
-    });
-
-    test("given a horizontal ray passing through the origin to the left", () => {
```

Comment:
This removed test was testing an invalid case. I'm okay leaving that behavior undefined.